app/models/accounting/quickbooks/updater.rb

```diff
@@ -47,9 +47,16 @@ def update(loan = nil)
         # but that condition is measured in days, not seconds, so this small a difference shouldn't matter.
         qb_connection.update_attribute(:last_updated_at, Time.now)
 
-        if loan
-          update_ledger(loan)
-          InterestCalculator.new(loan).recalculate
+        if loans
+          # check if loan is one object or multiple
+          [loans] if loans.is_a? Loan
```

Comment:
Look closely at this line, @arisah-ijeoma . It's not doing anything. You need to add `loans = ` at the beginning. 

Please also add to the comment at the top of this method noting that its argument can be either a single loan or a collection or array of loans.

Reply:
I'd also check why the test suite didn't catch this, if we're stubbing too much of this updater or not testing both types of input it would be nice to have a `TODO` to find some way to stub less of this workflow/add more unit tests.

Reply:
ah yes